packages/slate-react/src/plugins/after.js

```diff
@@ -245,6 +244,20 @@ function AfterPlugin(options = {}) {
     div.setAttribute('contenteditable', true)
     div.style.position = 'absolute'
     div.style.left = '-9999px'
+
+    // The snippet below is from `clipboard.js`
+    // Prevent zooming on iOS
+    div.style.fontSize = '12pt'
+    // Reset box model
+    div.style.border = '0'
+    div.style.padding = '0'
+    div.style.margin = '0'
```

Comment:
Do we need all of these tangential things that don't seem related to the scroll issue?

Reply:
The box dimensions interact with the 'top' attribute. It won't be a problem unless someone has all divs default to have margin near to the current window height. 

What do you think?

Reply:
In that case, for any that are necessary, let's put them all under the same `COMPAT:` comment so that it's clear. And any that aren't we remove

Reply:
I will remove:

```
    // Prevent zooming on iOS
    div.style.fontSize = '12pt'
```

and add the rest under COMPAT.